### ISSUE TITLE
模型的预训练截至到什么时间的？

### ISSUE THREAD

**Issue by loyedison92** [shiyu-coder/Kronos#204]
模型的预训练截至到什么时间的？

**Comment by shiyu-coder** [comment]
The cut-off date for the training set is February 2024 (inclusive). The validation set spans from March to May 2024 (inclusive), and the out-of-sample period begins in June 2024.
